### ISSUE TITLE
Node.js appliations in NGINX Unit failed to start due to missing `loader.js`.

### ISSUE THREAD

**Issue by krautcat** [nginx/unit#556]
I am trying to dockerize Node.js application.

There is my Dockerfile:

```Dockerfile
FROM nginx/unit:1.24.0-node15

WORKDIR /tmp

RUN apt-get update \
    && apt-get install --yes node-gyp

COPY odisharia.ru /var/www/odisharia.ru
WORKDIR /var/www/odisharia.ru

RUN npm install --save \
    && npm install --save --unsafe-perm unit-http \
    && npm run build

EXPOSE 80
EXPOSE 443
```

Building of my image goes ok, except this message:

```
Step 6/8 : RUN npm install --save     && npm install --save --unsafe-perm unit-http     && npm run build                                                                                                             
 ---> Running in 6be6dcf62bb4                                                                                                                                                                                        
                                                                                                                                                                                                                     
added 244 packages, and audited 245 packages in 13s                                                                                                                                                                  
                                                                                                                                                                                                                     
found 0 vulnerabilities                                                                                                                                                                                              
npm notice                                                                                                                                                                                                           
npm notice New minor version of npm available! 7.7.6 -> 7.16.0                                                                                                                                                       
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.16.0>                                                                                                                                              
npm notice Run `npm install -g npm@7.16.0` to update!                                                                                                                                                                
npm notice                                                                                                                                                                                                           
npm WARN EBADENGINE Unsupported engine {                                                                                                                                                                             
npm WARN EBADENGINE   package: '@sveltejs/kit@1.0.0-next.112',                                                                                                                                                       
npm WARN EBADENGINE   required: { node: '^12.20 || ^14.13.1 || >= 16' },                                                                                                                                             
npm WARN EBADENGINE   current: { node: 'v15.14.0', npm: '7.7.6' }                                                                                                                                                    
npm WARN EBADENGINE }                                                                                                                                                                                                
                                                                                                                                                                                                                     
added 1 package, and audited 246 packages in 3s                                                                                                                                                                      
                                                                                                                                                                                                                     
found 0 vulnerabilities
```

Then I try to start container based on this image and NGINX Unit failed to apply configuration and start application.

Here is my `config.json`:

```json
{
    "listeners": {
        "*:443": {
            "pass": "routes/odisharia_ru",
            "tls": {
                "certificate": "odisharia.ru"
            }
        }
    },

    "routes": {
        "odisharia_ru": [
            {
                "match": {
                    "host": "odisharia.ru",
                    "scheme": "https"
                },
                "action": {
                    "pass": "applications/main"
                }
            }
        ]
    },

    "applications": {
        "main": {
            "type": "external",
            "executable": "/usr/bin/env",
            "working_directory": "/var/www/odisharia.ru",
            "arguments": [
                "node", 
                "--loader",
                "unit-http/loader.mjs",
                "--require",
                "unit-http/loader",
                "build"
            ],
            "environment": {
                "HOST": "127.0.0.1",
                "PORT": "443"   
            }
        }
    },

    "access_log": "/var/log/access.log"
}
```

I get this message according to logs:

```
/usr/local/bin/docker-entrypoint.sh: Applying configuration /docker-entrypoint.d/config.json
2021/06/05 13:04:40 [info] 25#25 "main" application started
node:internal/modules/cjs/loader:927
  throw err;
  ^

Error: Cannot find module 'unit-http/loader'
Require stack:
- internal/preload
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at Module._preloadModules (node:internal/modules/cjs/loader:1251:12)
    at loadPreloadModules (node:internal/bootstrap/pre_execution:455:5)
    at prepareMainThreadExecution (node:internal/bootstrap/pre_execution:74:3)
    at node:internal/main/run_main_module:7:1 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'internal/preload' ]
}
2021/06/05 13:04:40 [notice] 12#12 process 25 exited with code 1
2021/06/05 13:04:40 [warn] 16#16 failed to start application "main"
2021/06/05 13:04:40 [alert] 16#16 failed to apply new conf
/usr/local/bin/docker-entrypoint.sh: Error: HTTP response status code is '500'
{
        "error": "Failed to apply new configuration."
}
```

I tried to do it without Docker ang got the same error.

What am I doing wrong?

**Comment by mar0x** [comment]
Sorry, we miss the file in `package.json`. The fix is ready, we will release it soon. For now, consider using local fixed package (copy attached [unit-http-1.24.0.tar.gz](https://github.com/nginx/unit/files/6629624/unit-http-1.24.0.tar.gz) into docker image and install there).

**Comment by mar0x** [comment]
The issue should be fixed in `1.24.1`. Please update `unit-http` from npm. Note: other Unit packages is not updated, only npm package affected.
